addons/binding/org.openhab.binding.boschindego/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BoschIndego Binding
+Bundle-SymbolicName: org.openhab.binding.boschindego;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/bosch-indego-controller-lib-0.8.jar,
+ lib/httpclient-4.5.2.jar,
+ lib/httpcore-4.4.4.jar,
+ lib/commons-codec-1.9.jar,
```

Comment:
The commons-logging is needed because it is an dependency of the apache http client. Its not used. I used the logging lib included in openHAB. Removing the version tags solved the problems with the different http requests. 

>